extensions/wikia/ImageReview/ImageReviewImageStatus.php

```diff
@@ -50,8 +50,37 @@ function efImageReviewDisplayStatus( $imagePage, &$html ) {
 		);
 
 		if ( false === $imgCurState ) {
+			/**
+			 * If the file is older than 1 hour - send it to ImageReview
+			 * since it's probably been restored, and is not just a fresh file.
+			 */
+			$lastTouched = new DateTime( $imagePage->getRevisionFetched()->getTimestamp() );
+			$now = new DateTime();
+			if ( $lastTouched < $now->modify( '-1 hour' ) ) {
+				$scribeEventProducer = new ScribeEventProducer( 'edit' );
+				$user = $imagePage->getContext()->getUser();
```

Comment:
This is from the RequestContext, so would be the user viewing the image page, not the user who uploaded it, right? Should we instead get the user who uploaded the file to insert into the ImageReview table?